src/test/java/AccountTest.java

```diff
@@ -138,4 +151,37 @@ public void testGetAccountSheetsRowMethod() {
 
     assertEquals(expectedResponse, actualResponse);
   }
+
+  /** Confirm that edits made to an account are applied correctly */
+  @Test
+  public void testUpdateExistingAccount() {
+    HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
+    Mockito.when(request.getParameter(FormIdNames.VENDOR_ID))
```

Comment:
Could you add a comment on what Mockito actually does?

Reply:
yep, good call

Reply:
Nice test. It would be cool if we could test all of the servlets like this, but that's a bit out of scope at this point.